openedx/adg/lms/applications/__init__.py

```diff
@@ -1,5 +1,42 @@
 """
-init file for applications
-"""
+This app adds the ability to submit, review and manage applications for users and managers.
+
+Application hub
+------------
+ADG provides an application hub where users (learners) can go and submit application for review.
+Each user can submit only one application. Application submission process on Application hub
+consists of two mandatory steps i.e. prerequisite courses and written application.
+
+Prerequisite courses
+------------
+Application hub shows (configured) courses in its dashboard which are mandatory to pass in order to submit
+the application. Each course is available in two languages i.e Arabic and English.
+
+Written application
+------------
+Written application is another mandatory step to submit the application. It asks the applicant few questions from
+various sections including, contact information, education and business line/unit to apply to. User can upload CV
+and cover letter as well. All the data is saved in database.
 
+Admin Dashboard
+------------
+With each business line there is a Django user group attached, which is used to create business unit specific
+admins, though there are ADG admins as well. All types of admin are listed below:
+
+1. Django admin (Super user)
+1. ADG admin
+3. Business unit (BU) admin
+
+ADG and BU admins can access limited but related models only, via admin dashboard, which is a subset of Django
+admin view. ADG admins can view and manage all applications, however, BU admins can only manage applications
+related to specific business line/unit.
+
+Multilingual courses
+------------
+Each course is available in two languages Arabic and English. To manage this, multilingual course group is used.
```

Comment:
We have build a generic functionality so shouldn't document it as "could exist in two languages" we should make the documentation generic too.